### ISSUE TITLE
503-Service unavailable error for getDadJoke

### ISSUE THREAD

**Issue by LManasa2395** [Saurabh3333/give-me-a-joke#15]
When we try to execute getDadJoke function
`jokes.getRandomDadJoke((joke) => console.log(joke));`

 we are receiving the below error:
```
Error: Request failed with status code 503
    at createError (D:\Udemy Courses\WDB\Code\WDB\Jokester\node_modules\axios\lib\core\createError.js:16:15)
    at settle (D:\Udemy Courses\WDB\Code\WDB\Jokester\node_modules\axios\lib\core\settle.js:17:12)
    at IncomingMessage.handleStreamEnd (D:\Udemy Courses\WDB\Code\WDB\Jokester\node_modules\axios\lib\adapters\http.js:236:11)
    at IncomingMessage.emit (node:events:406:35)
    at endReadableNT (node:internal/streams/readable:1329:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  config: {
    url: 'https://icanhazdadjoke.com/',
    headers: { Accept: 'application/json', 'User-Agent': 'axios/0.19.2' },
    transformRequest: [ [Function: transformRequest] ],
    transformResponse: [ [Function: transformResponse] ],
    timeout: 0,
    adapter: [Function: httpAdapter],
    xsrfCookieName: 'XSRF-TOKEN',
    xsrfHeaderName: 'X-XSRF-TOKEN',
    maxContentLength: -1,
    validateStatus: [Function: validateStatus],
    method: 'get',
    data: undefined
  },
  request: <ref *1> ClientRequest {
    _events: [Object: null prototype] {
      abort: [Function (anonymous)],
      aborted: [Function (anonymous)],
      error: [Function (anonymous)],
      socket: [Function (anonymous)],
      timeout: [Function (anonymous)],
      prefinish: [Function: requestOnPrefinish]
    },
    _eventsCount: 6,
    _maxListeners: undefined,
    outputData: [],
    outputSize: 0,
    writable: true,
    destroyed: false,
    _last: true,
    chunkedEncoding: false,
    shouldKeepAlive: false,
    _defaultKeepAlive: true,
    useChunkedEncodingByDefault: false,
    sendDate: false,
    _removedConnection: false,
    _removedContLen: false,
    _removedTE: false,
    _contentLength: 0,
    _hasBody: true,
    _trailer: '',
    finished: true,
    _headerSent: true,
    _closed: false,
    socket: TLSSocket {
      _tlsOptions: [Object],
      _secureEstablished: true,
      _securePending: false,
      _newSessionPending: false,
      _controlReleased: true,
      secureConnecting: false,
      _SNICallback: null,
      servername: 'icanhazdadjoke.com',
      alpnProtocol: false,
      authorized: true,
      authorizationError: null,
      encrypted: true,
      _events: [Object: null prototype],
      _eventsCount: 10,
      connecting: false,
      _hadError: false,
      _parent: null,
      _host: 'icanhazdadjoke.com',
      _readableState: [ReadableState],
      _maxListeners: undefined,
      _writableState: [WritableState],
      allowHalfOpen: false,
      _sockname: null,
      _pendingData: null,
      _pendingEncoding: '',
      server: undefined,
      _server: null,
      ssl: [TLSWrap],
      _requestCert: true,
      _rejectUnauthorized: true,
      parser: null,
      _httpMessage: [Circular *1],
      [Symbol(res)]: [TLSWrap],
      [Symbol(verified)]: true,
      [Symbol(pendingSession)]: null,
      [Symbol(async_id_symbol)]: 6,
      [Symbol(kHandle)]: [TLSWrap],
      [Symbol(kSetNoDelay)]: false,
      [Symbol(lastWriteQueueSize)]: 0,
      [Symbol(timeout)]: null,
      [Symbol(kBuffer)]: null,
      [Symbol(kBufferCb)]: null,
      [Symbol(kBufferGen)]: null,
      [Symbol(kCapture)]: false,
      [Symbol(kBytesRead)]: 0,
      [Symbol(kBytesWritten)]: 0,
      [Symbol(connect-options)]: [Object],
      [Symbol(RequestTimeout)]: undefined
    },
    _header: 'GET / HTTP/1.1\r\n' +
      'Accept: application/json\r\n' +
      'User-Agent: axios/0.19.2\r\n' +
      'Host: icanhazdadjoke.com\r\n' +
      'Connection: close\r\n' +
      '\r\n',
    _keepAliveTimeout: 0,
    _onPendingData: [Function: nop],
    agent: Agent {
      _events: [Object: null prototype],
      _eventsCount: 2,
      _maxListeners: undefined,
      defaultPort: 443,
      protocol: 'https:',
      options: [Object: null prototype],
      requests: [Object: null prototype] {},
      sockets: [Object: null prototype],
      freeSockets: [Object: null prototype] {},
      keepAliveMsecs: 1000,
      keepAlive: false,
      maxSockets: Infinity,
      maxFreeSockets: 256,
      scheduling: 'lifo',
      maxTotalSockets: Infinity,
      totalSocketCount: 1,
      maxCachedSessions: 100,
      _sessionCache: [Object],
      [Symbol(kCapture)]: false
    },
    socketPath: undefined,
    method: 'GET',
    maxHeaderSize: undefined,
    insecureHTTPParser: undefined,
    path: '/',
    _ended: true,
    res: IncomingMessage {
      _readableState: [ReadableState],
      _events: [Object: null prototype],
      _eventsCount: 3,
      _maxListeners: undefined,
      socket: [TLSSocket],
      httpVersionMajor: 1,
      httpVersionMinor: 1,
      httpVersion: '1.1',
      complete: true,
      rawHeaders: [Array],
      rawTrailers: [],
      aborted: false,
      upgrade: false,
      url: '',
      method: null,
      statusCode: 503,
      statusMessage: 'Service Temporarily Unavailable',
      client: [TLSSocket],
      _consuming: true,
      _dumped: false,
      req: [Circular *1],
      responseUrl: 'https://icanhazdadjoke.com/',
      redirects: [],
      [Symbol(kCapture)]: false,
      [Symbol(kHeaders)]: [Object],
      [Symbol(kHeadersCount)]: 32,
      [Symbol(kTrailers)]: null,
      [Symbol(kTrailersCount)]: 0,
      [Symbol(RequestTimeout)]: undefined
    },
    aborted: false,
    timeoutCb: null,
    upgradeOrConnect: false,
    parser: null,
    maxHeadersCount: null,
    reusedSocket: false,
    host: 'icanhazdadjoke.com',
    protocol: 'https:',
    _redirectable: Writable {
      _writableState: [WritableState],
      _events: [Object: null prototype],
      _eventsCount: 2,
      _maxListeners: undefined,
      _options: [Object],
      _redirectCount: 0,
      _redirects: [],
      _requestBodyLength: 0,
      _requestBodyBuffers: [],
      _onNativeResponse: [Function (anonymous)],
      _currentRequest: [Circular *1],
      _currentUrl: 'https://icanhazdadjoke.com/',
      [Symbol(kCapture)]: false
    },
    [Symbol(kCapture)]: false,
    [Symbol(kNeedDrain)]: false,
    [Symbol(corked)]: 0,
    [Symbol(kOutHeaders)]: [Object: null prototype] {
      accept: [Array],
      'user-agent': [Array],
      host: [Array]
    }
  },
  response: {
    status: 503,
    statusText: 'Service Temporarily Unavailable',
    headers: {
      date: 'Wed, 04 Aug 2021 14:49:38 GMT',
      'content-type': 'text/html; charset=UTF-8',
      'transfer-encoding': 'chunked',
      connection: 'close',
      'x-frame-options': 'SAMEORIGIN',
      'permissions-policy': 'accelerometer=(),autoplay=(),camera=(),clipboard-read=(),clipboard-write=(),fullscreen=(),geolocation=(),gyroscope=(),hid=(),interest-cohort=(),magnetometer=(),microphone=(),payment=(),publickey-credentials-get=(),screen-wake-lock=(),serial=(),sync-xhr=(),usb=()',
      'cache-control': 'private, max-age=0, no-store, no-cache, must-revalidate, post-check=0, pre-check=0',
      expires: 'Thu, 01 Jan 1970 00:00:01 GMT',
      'expect-ct': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"',
      'report-to': '{"endpoints":[{"url":"https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=jG%2FsJJS7ym3gB3jSWzAy35QXsODGq94XWtMErj%2B63NKJoC%2FJL0%2Fi1ew7KvPfft3SkttUZfOYnDktxfR9sAcZmjGMkdGGbcp6zuPNtcikEmwlmYWKmyPITewjWeCzxEi60Y5qMm4%3D"}],"group":"cf-nel","max_age":604800}',
      nel: '{"report_to":"cf-nel","max_age":604800}',
      'strict-transport-security': 'max-age=15552000; includeSubDomains',
      'x-content-type-options': 'nosniff',
      server: 'cloudflare',
      'cf-ray': '679897ef79534b0b-HYD',
      'alt-svc': 'h3-27=":443"; ma=86400, h3-28=":443"; ma=86400, h3-29=":443"; ma=86400, h3=":443"; ma=86400'
    },
    config: {
      url: 'https://icanhazdadjoke.com/',
      headers: [Object],
      transformRequest: [Array],
      transformResponse: [Array],
      timeout: 0,
      adapter: [Function: httpAdapter],
      xsrfCookieName: 'XSRF-TOKEN',
      xsrfHeaderName: 'X-XSRF-TOKEN',
      maxContentLength: -1,
      validateStatus: [Function: validateStatus],
      method: 'get',
      data: undefined
    },
    request: <ref *1> ClientRequest {
      _events: [Object: null prototype],
      _eventsCount: 6,
      _maxListeners: undefined,
      outputData: [],
      outputSize: 0,
      writable: true,
      destroyed: false,
      _last: true,
      chunkedEncoding: false,
      shouldKeepAlive: false,
      _defaultKeepAlive: true,
      useChunkedEncodingByDefault: false,
      sendDate: false,
      _removedConnection: false,
      _removedContLen: false,
      _removedTE: false,
      _contentLength: 0,
      _hasBody: true,
      _trailer: '',
      finished: true,
      _headerSent: true,
      _closed: false,
      socket: [TLSSocket],
      _header: 'GET / HTTP/1.1\r\n' +
        'Accept: application/json\r\n' +
        'User-Agent: axios/0.19.2\r\n' +
        'Host: icanhazdadjoke.com\r\n' +
        'Connection: close\r\n' +
        '\r\n',
      _keepAliveTimeout: 0,
      _onPendingData: [Function: nop],
      agent: [Agent],
      socketPath: undefined,
      method: 'GET',
      maxHeaderSize: undefined,
      insecureHTTPParser: undefined,
      path: '/',
      _ended: true,
      res: [IncomingMessage],
      aborted: false,
      timeoutCb: null,
      upgradeOrConnect: false,
      parser: null,
      maxHeadersCount: null,
      reusedSocket: false,
      host: 'icanhazdadjoke.com',
      protocol: 'https:',
      _redirectable: [Writable],
      [Symbol(kCapture)]: false,
      [Symbol(kNeedDrain)]: false,
      [Symbol(corked)]: 0,
      [Symbol(kOutHeaders)]: [Object: null prototype]
    },
    data: '<!DOCTYPE HTML>\n' +
      '<html lang="en-US">\n' +
      '<head>\n' +
      '  <meta charset="UTF-8" />\n' +
      '  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />\n' +
      '  <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1" />\n' +
      '  <meta name="robots" content="noindex, nofollow" />\n' +
      '  <meta name="viewport" content="width=device-width,initial-scale=1" />\n' +
      '  <title>Just a moment...</title>\n' +
      '  <style type="text/css">\n' +
      '    html, body {width: 100%; height: 100%; margin: 0; padding: 0;}\n' +
      '    body {background-color: #ffffff; color: #000000; font-family:-apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, "Helvetica 
Neue",Arial, sans-serif; font-size: 16px; line-height: 1.7em;-webkit-font-smoothing: antialiased;}\n' +
      '    h1 { text-align: center; font-weight:700; margin: 16px 0; font-size: 32px; color:#000000; line-height: 1.25;}\n' +
      '    p {font-size: 20px; font-weight: 400; margin: 8px 0;}\n' +
      '    p, .attribution, {text-align: center;}\n' +
      '    #spinner {margin: 0 auto 30px auto; display: block;}\n' +
      '    .attribution {margin-top: 32px;}\n' +
      '    @keyframes fader     { 0% {opacity: 0.2;} 50% {opacity: 1.0;} 100% {opacity: 0.2;} }\n' +
      '    @-webkit-keyframes fader { 0% {opacity: 0.2;} 50% {opacity: 1.0;} 100% {opacity: 0.2;} }\n' +
      '    #cf-bubbles > .bubbles { animation: fader 1.6s infinite;}\n' +
      '    #cf-bubbles > .bubbles:nth-child(2) { animation-delay: .2s;}\n' +
      '    #cf-bubbles > .bubbles:nth-child(3) { animation-delay: .4s;}\n' +
      '    .bubbles { background-color: #f58220; width:20px; height: 20px; margin:2px; border-radius:100%; display:inline-block; }\n' +
      '    a { color: #2c7cb0; text-decoration: none; -moz-transition: color 0.15s ease; -o-transition: color 0.15s ease; -webkit-transition: color 0.15s ease; transition: color 0.15s ease; }\n' +
      '    a:hover{color: #f4a15d}\n' +
      '    .attribution{font-size: 16px; line-height: 1.5;}\n' +
      '    .ray_id{display: block; margin-top: 8px;}\n' +
      '    #cf-wrapper #challenge-form { padding-top:25px; padding-bottom:25px; }\n' +
      '    #cf-hcaptcha-container { text-align:center;}\n' +
      '    #cf-hcaptcha-container iframe { display: inline-block;}\n' +
      '  </style>\n' +
      '\n' +
      '      <meta http-equiv="refresh" content="3">\n' +
      '  <script type="text/javascript">\n' +
      '    //<![CDATA[\n' +
      '    (function(){\n' +
      '      \n' +
      '      window._cf_chl_opt={\n' +
      '        cvId: "2",\n' +
      '        cType: "non-interactive",\n' +
      '        cNounce: "79770",\n' +
      '        cRay: "679897ef79534b0b",\n' +
      '        cHash: "6d329fcd7a7e162",\n' +
      '        cFPWv: "g",\n' +
      '        cTTimeMs: "1000",\n' +
      '        cRq: {\n' +
      '          ru: "aHR0cHM6Ly9pY2FuaGF6ZGFkam9rZS5jb20v",\n' +
      '          ra: "YXhpb3MvMC4xOS4y",\n' +
      '          rm: "R0VU",\n' +
      '          d: "Xj9Gw7hwRfXS3Dxm99n27sRY2BoTDvGzoXjCOtfVXcf1j5Pm7sVppjLLBW4POA5kdzTI7FAK5Peqze8fFee9mZ9NaVK3s4IN7EkYXMbMbNDTpmwdizKRtWvDCbQv10m1jZpu4GNF/fF8eetVmHGrXoIn3MdYbreU28I22nFlrrddaL20HMpxf8JQ9VMAqA2k9goVxXNFbx245+36kFuXix4Zn8ssvtq5wqfanfP9//3CAkXXRY8U+MpfOXa2MjsytoYyy65vhK8UND8ORXW1ZzMWI8evxdSX5qp6IHTWk3EoxcgR4eUJym3SMZE/7N8SpC+429yx0lmaIzxanIrzsIWfK8oXQStcDSYfRPCmXVH2HYypJawh8n18WrrvWi8RPK6gKKRE26Wm6ghdiUgx4P6pmTNy6QvowwQyBGh0DpjfyWQg4xbKo4LhG/MZx0NXwsJ6FK92/6a4lDv+dMK6Y021uBa4JK9MgKQ0gGMWa4zcrcRJEE00od9H5daJ0beoIVk9rpmh71JRAloYlUly99qerPolH9KQvajYQiyPeU/rnYtzuVu5w4msYW+6oC+nDPQuG/C9p8SM7WewSIE//hopTXioPisDTILTmZlpF0zjrO41gOsAofhpMsHZ1B/U0IBlEB/+9CBbtetzQ7zMIYVI+jwHehUnQo7KMBPv1qtgT4EAxZAliULtZ3shphhAsdA2czbdzclE7xJFcZaEsWWDuxQQaI+7GMsk4P6illyUTiXX/Stf9xb1+oI1Y2S2eqaKrDh8bLqdEpmdNsPQnn7P/sScyeFrFZUmUJRUXNFWPodhMhcREUFcuheXWy51yIUSDHp39SgPM7IVcFOT8W5UXq8TTUL1UxdtKvCJ0Ws=",\n' +
      '          t: "MTYyODA4ODU3OC40NzgwMDA=",\n' +
      '          m: "zBiFHQH5wDUBDSYes9uN2h46IRVVb6QSHrH1XT1EWds=",\n' +
      '          i1: "5NcLMqNK/n1KRC5zog+X5Q==",\n' +
      '          i2: "N/yfgvmF5V7tL1VhnwLFHg==",\n' +
      '          zh: "VfquuLa71QtZvymjqUyepHo83ERAHPAuEz8zYqfNN1Q=",\n' +
      '          uh: "BJqLy5IxKHhex0MdF9TScP+4XqNT/8AmCW7UXDrTzac=",\n' +
      '          hh: "otN/W7hNWHlBPhryZtdhMvQG/CHyD4RT7drKXVkpx4E=",\n' +
      '        }\n' +
      '      }\n' +
      '      window._cf_chl_enter = function(){window._cf_chl_opt.p=1};\n' +
      '      \n' +
      '    })();\n' +
      '    //]]>\n' +
      '  </script>\n' +
      '  \n' +
      '\n' +
      '</head>\n' +
      '<body>\n' +
      '  <table width="100%" height="100%" cellpadding="20">\n' +
      '    <tr>\n' +
      '      <td align="center" valign="middle">\n' +
      '          <div class="cf-browser-verification cf-im-under-attack">\n' +
      '  <noscript>\n' +
      '    <h1 data-translate="turn_on_js" style="color:#bd2426;">Please turn JavaScript on and reload the page.</h1>\n' +
      '  </noscript>\n' +
      '  <div id="cf-content" style="display:none">\n' +
      '    \n' +
      '    <div id="cf-bubbles">\n' +
      '      <div class="bubbles"></div>\n' +
      '      <div class="bubbles"></div>\n' +
      '      <div class="bubbles"></div>\n' +
      '    </div>\n' +
      '    <h1><span data-translate="checking_browser">Checking your browser before accessing</span> icanhazdadjoke.com.</h1>\n' +
      '    \n' +
      '    <div id="no-cookie-warning" class="cookie-warning" data-translate="turn_on_cookies" style="display:none">\n' +
      '      <p data-translate="turn_on_cookies" style="color:#bd2426;">Please enable Cookies and reload the page.</p>\n' +
      '    </div>\n' +
      '    <p data-translate="process_is_automatic">This process is automatic. Your browser will redirect to your requested content shortly.</p>\n' +
      '    <p data-translate="allow_5_secs" id="cf-spinner-allow-5-secs" >Please allow up to 5 seconds&hellip;</p>\n' +
      '    <p data-translate="redirecting" id="cf-spinner-redirecting" style="display:none">Redirecting&hellip;</p>\n' +
      '  </div>\n' +
      '   \n' +
      '  <form class="challenge-form" id="challenge-form" action="/?__cf_chl_jschl_tk__=pmd_2d13b194a9b0879b584538be483c68d8e8e30f7a-1628088578-0-gqNtZGzNAg2jcnBszQci" method="POST" enctype="application/x-www-form-urlencoded">\n' +
      '    <input type="hidden" name="md" value="f80ccf5580c67006baf2d46834e175eda962786d-1628088578-0-AR_i-gfsn0AfGzrAW2h_PoDquZ-yH8OF6plulwIIVk0MNW4rf1K2xkNTaZWI9mhC1lN4MzzA4B2TMMLUMyN1VKgB5DTlICKFAWN_MShnqn8Kr1OhA2YfATv97QbT-BuEuNTcAZOiQzimDFLOWpuvrcJNtqGyghLbU4rdtFcrUKdtoINI2FrZ9oyeyDfkpnSTZOIXKu0vrwY3ZtOcLgXivWWAwOy-ekz3Tpgz_prBWxaKpDcH8s_sMGPCE5diCKk5o6outa6_cfEC-LFcCPWUB_B0UBxx9mxaYe5SFVNyKjMqHVu_qYeLEE9cy1RlQ1pnQp1aV14HkVzP4AGls2wdLDGmeVH_If-bVjKKTikhYvBIgDvn22XDhf9jFpmfe0ijLnvwsdqMfegfc_FCgb_PlbU1rBMHoLzzA96s6AnKgl5DXuW11VD3T2mie6isvgRS9d_4TH_6s5eRTWCrCznbIjg" />\n' +
      '    <input type="hidden" name="r" value="597c8091f456c1992db77dd960620dae137b1bd7-1628088578-0-AeecW3kWOTgmeSYJxbYfHTjbi0Ef7v2/lAuE3dDB2LLdHk0K49iym1ul4WG6uQROH8kOQzfaEj85W6YbGcEffSEEbT2zQU5ig49iOTCgz6n9BgclP9mVHKo0SSbkTgp3xLRvkMmUOAHXI012BqioNDhy9O6VzW50RSanrscjOgwMmj/QGNssf1V6YtI6/LgEt8r6D3yuCGqzDCoULztGP9fhqi9oxd1tIEarxLXpUEvnJlUpQQqUWNkgrgszG88xx1pfkNliUrjB7aq+IAVtaNVFppZH2yupEbYDwiGAFL8M5nnfGV6Ql7EQgmtRIFJ+1eXaJVr65M4NfBpb+XaPvmwuDrfbBLN4KFZWNzh2QWVZ/T3Tj3BxtSShdcCfyj8twiFpiLEUinwGbFVVRi/5k4fsfU7A3HsGoTeCNs1q/MRDOicAz/2iCumbN9vIiBsK+q3n4KPfyd2XWv7RulQUj6r62/E1BG5t5ZZqaAoI/qZCBuSAYtm7FyqINloFrR2wefx1HISVy196UdkSU0KdsBIg9dy/kNM5HKIOf6iAlBvoVJQMA6pBHWWcgp7I5zJFh6aFQeol4uBhGCxwPfeG2IUX/63VsA72tNQtinX2JMjd6vkFlkVsfNDucnXN1TbcBjU748o94bUr2dpeQp+uHKN8nMUKCNO/cnWTQR/PgFaOoh7y9kuSXeIDpp7TsJy1DWEDp7rAF42JcLwk90HYSkWWo62tx9uLTc0C18Bz8+i3u6kTjbjwoQX05TZAonKHvvBmhhp9IVPQQtrIo+4yFalldxE8XyMSA9agSuNf98oxgq2ZzuVNpYcv8XgoL74EEQEcFnO8bWMO0I3kOVdFRQ6tfTLWF1qVnJiCo49GOvgNbExI5KbpT8gFIgay0rX/1RhJimKQzHmosIj0Kwnx82E4BWiSeWWjVdiRkESjFsMqDwpeOHNC8VE11nmvL/JajRkxGPx/0X0UDIRbY9GOtIKJl369+M6wxy0RfSLSFwz1XOz6DGMOi+8y31P0u8eTNbnH9D46cw/uFqZFXcBsD8RMU9FQX8u3Dv+yRxu7OqnY27X3NCa4KTlgelQsi1G7XHh7HIWXHZXqClFs0t1sbyaRGC15EGBUEUCez1v0f2nznO1o4o1whQLiMfKdRo4wZQ9r0pLVHX7uYta2T32/HAKqZOFw6vyxioEqRB+uxUiUYCJt6JVG2HGZwjAr9Iu+61LEYj5aW+qKCNfetn5d2yFxRLXPkiMI4MwPpjQdCq5WX1gToPi4QK+reYkR9S1J/tXYJups86ZY8LaW1IViT60nNGadJR/s1zpghSzKXKXTtm6cbKgOQLKxXPvvH4u6HqvxxV4mYCPzJevjAJzNr0Z/gUmR3XHxkLJIlKoGdM7iAVw1OAqJnb9WSOSpdPQn+pdd4EN+6H5+b9egUxkQu6lCALNKpbGQR9MQ2qLmTxUd/wAZO3I0aPGDgATNmUPh6V8fAROoMVcBsGipoTX+o8gD2Ew5MU3Qs4QIbwZzUO+LUXwN5SDd1jxck5lx6cj7T21TUiQgML8gnYTwAuQTL6N6gKzrmaKxAejKiOWHv5wv4HaB0I5gd+BxOnahOTgGFsn72TuQR1myrowOKxlRBhZ4SSaInzf7VrCSSwZsexJI8+9L4V1LP99e1xMLdSx1dLnyJ/fa5Y939KGWugZ1ERPn+POBHiNwnoPIiDz1m5n+rM1gG3vQTNpWeMA32PgJAqn9Gk6WBplWB655iiu6Nf93SY8C1ZwK7LInmYdGVaZh"/>\n' +
      '    <input type="hidden" value="b604ba207922609ad6da96abe3c7f602" id="jschl-vc" name="jschl_vc"/>\n' +
      '    <!-- <input type="hidden" value="" id="jschl-vc" name="jschl_vc"/> -->\n' +
      '    <input type="hidden" name="pass" value="1628088579.478-TWxIHP6BsR"/>\n' +
      '    <input type="hidden" id="jschl-answer" name="jschl_answer"/>\n' +
      '  </form>\n' +
      '     \n' +
      '    <script type="text/javascript">\n' +
      '      //<![CDATA[\n' +
      '      (function(){\n' +
      "          var a = document.getElementById('cf-content');\n" +
      "          a.style.display = 'block';\n" +
      '          var isIE = /(MSIE|Trident\\/|Edge\\/)/i.test(window.navigator.userAgent);\n' +
      "          var trkjs = isIE ? new Image() : document.createElement('img');\n" +
      '          trkjs.setAttribute("src", "/cdn-cgi/images/trace/jschal/js/transparent.gif?ray=679897ef79534b0b");\n' +
      '          trkjs.id = "trk_jschal_js";\n' +
      '          trkjs.setAttribute("alt", "");\n' +
      '          document.body.appendChild(trkjs);\n' +
      "          var cpo=document.createElement('script');\n" +
      "          cpo.type='text/javascript';\n" +
      '          cpo.src="/cdn-cgi/challenge-platform/h/g/orchestrate/jsch/v1?ray=679897ef79534b0b";\n' +
      "          document.getElementsByTagName('head')[0].appendChild(cpo);\n" +
      '        }());\n' +
      '      //]]>\n' +
      '    </script>\n' +
      '  \n' +
      '\n' +
      '  \n' +
      `  <div id="trk_jschal_nojs" style="background-image:url('/cdn-cgi/images/trace/jschal/nojs/transparent.gif?ray=679897ef79534b0b')"> </div>\n` +
      '</div>\n' +
      '\n' +
      '          <a href="https://noabcla.com/cultured.php?more=18"><!-- table --></a>\n' +
      '          <div class="attribution">\n' +
      '            DDoS protection by <a rel="noopener noreferrer" href="https://www.cloudflare.com/5xx-error-landing/" target="_blank">Cloudflare</a>\n' +
      '            <br />\n' +
      '            <span class="ray_id">Ray ID: <code>679897ef79534b0b</code></span>\n' +
      '          </div>\n' +
      '      </td>\n' +
      '     \n' +
      '    </tr>\n' +
      '  </table>\n' +
      '</body>\n' +
      '</html>\n'
  },
  isAxiosError: true,
  toJSON: [Function (anonymous)]
}
```

**Comment by Saurabh3333** [comment]
Looks like a local issue. Just checked, it's working fine for me. Can you please try it with some other project?

**Comment by nax3t** [comment]
Not a local issue, the https://icanhazdadjoke.com/ site isn't playing nice with Axios (updated to 0.21.1 to mitigate risk). At least not with the configuration that you have set in index.js
I tried it with curl and it returns a joke, so the API is working.

**Comment by Saurabh3333** [comment]
Yes, you are right. `getRandomDadJoke (function(joke)` has some problems.

**Comment by nax3t** [comment]
Happy to help if you don't have the time, just let me know

**Comment by Saurabh3333** [comment]
Thanks, @nax3t
@LManasa2395 All Issues are resolved in v0.5.1. Please retry.
